privacyidea/api/resolver.py

```diff
@@ -55,30 +55,38 @@
 resolver_blueprint = Blueprint('resolver_blueprint', __name__)
 
 
-# ----------------------------------------------------------------------
-#
-#   Resolver methods
-#
-
 @resolver_blueprint.route('/', methods=['GET'])
+@resolver_blueprint.route('/<resolver>', methods=['GET'])
 @log_with(log)
-def get_resolvers():
+def get_resolvers(resolver=None):
     """
-    returns a json list of all resolver.
+    returns a json list of the specified resolvers.
+    The passwords of resolvers (e.g. Bind PW of the LDAP resolver or password of the
+    SQL resolver) will be returned as "__CENSORED__".
+    You can run a POST request to update the data and privacyIDEA will ignore the "__CENSORED__"
+    password or you can even run a testresolver.
 
+    :param resolver: the name of the resolver
+    :type resolver: basestring
```

Comment:
There is no builtin type `basestring` in Python 3. I would prefer just `str` because this covers most cases.
What do You think?